resolwe_bio/processes/differential_expression/edgeR.py

```diff
@@ -20,6 +21,33 @@
 )
 
 
+def prepare_expressions(inputs, count_filter, output_file):
+    """Concatenate expression matrices and filter.
+    
+    This function is a drop-in replacement for expressionmerge.py
+    and diffexp_filtering.R.
+    """
+    exps = []
+    for exp in inputs:
+        temp = pd.read_csv(exp, sep="\t", index_col=0)
+        temp.columns = [exp]
+        exps.append(temp)
+
+    exps = pd.concat(exps, axis=1)
+    # Make sure the order of columns corresponds to the order of input
+    # data.
+    exps = exps.loc[:, inputs]
+
+    exps = exps.dropna(axis=1)
+
+    # This is calculating the filter only on numeric columns, so excluding
+    # column Gene.
+    check_filter = exps.select_dtypes(include="number") >= count_filter
```

Comment:
I believe this should behave in the same way as implemented in deseq2 process - the **sum** of counts across all included samples should be above the set threshold. The way it is implemented here and also in the deprecated R script, all samples must have counts above the set threshold for the gene to be retained.

Reply:
Done. Perhaps we could also think about moving the `inputs.count_filter` to it's own group to match the deseq process.